Sources/qiskit/datastructures/Matrix+ColumnRowValueArray.swift

```diff
@@ -0,0 +1,28 @@
+// Copyright 2018 IBM RESEARCH. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// =============================================================================
+
+extension Matrix {
+    public func columnRowValueArray() -> [[Any]] {
```

Comment:
Used [to plot a state with method "city"](https://github.com/indisoluble/qiskit-sdk-swift/blob/master/Sources/qiskit/tools/visualization/Visualization.swift#L65).